packages/expo-router/src/global-state/router-store.tsx

```diff
@@ -177,7 +178,7 @@ export class RouterStore {
     return getRouteInfoFromState(
       (state: Parameters<typeof getPathFromState>[0], asPath: boolean) => {
         return getPathDataFromState(state, {
-          screens: [],
+          screens: {},
```

Comment:
This should be an `object`, but this never failed because

1) Its always overridden by `this.linking.config`
2) The types were previously incorrect (they used `object`, and an array satisfies that type as it is an object)